extensions/amp-story/0.1/amp-story-system-layer.js

```diff
@@ -314,6 +326,15 @@ export class SystemLayer {
     this.storeService_.dispatch(Action.TOGGLE_SHARE_MENU, !isOpen);
   }
 
+  /**
+   * Handles click events on the info button and toggles the info dialog.
+   * @private
+   */
+  onInfoClick_() {
+    const isOpen = this.storeService_.get(StateProperty.INFO_DIALOG_STATE);
+    this.storeService_.dispatch(Action.TOGGLE_INFO_DIALOG, !isOpen);
```

Comment:
Super nitty / discussion:

Could we build the domain info dialog only when the user clicks this button to open it? I believe the code would be easy to write, but there may be edge cases I don't know about.
Would calling ``this.build().then(/* Add class to display the dialog */)`` from the store info dialog state listener work?

Opening the dialog becoming asynchronous, we then might want to get this ``onInfoClick`` method a way to open the dialog only, to make sure a user clicking twice on this button doesn't hide the dialog back.

Reply:
I think it may be slightly more complicated due to the promise for the more info link.  Mind if I add in a follow up PR?

Reply:
Sounds good :)